### ISSUE TITLE
Adding 'delighters' to Wagtail

### ISSUE THREAD

**Issue by heymonkeyriot** [torchbox/wagtail#2356]
This relates loosely to conversations happening around login and the dashboard (e.g. #2342) but I think we can improve the tone of voice and introduce some 'delighters' to the product. At the moment Wagtail's messaging is fairly conservative with language and imagery across both the CMS and website. I think introducing a more upbeat, personal tone, especially around on-boarding and login, will better suit our goals with Wagtail.

Some obvious examples of what I mean
![image](https://cloud.githubusercontent.com/assets/11335309/13755921/5650eaaa-ea15-11e5-809e-8e8c89ec8ab3.png)

![image](https://cloud.githubusercontent.com/assets/11335309/13755926/5efa9f66-ea15-11e5-8600-a5dce9eff055.png)

I'm aware this is somewhat of a trend at the moment, and there's no point just jumping on a bandwagon for the sake of it, but speaking for myself when a product I'm using has a bit of 'personality' it enhances my experience of it. 

I'm conscious the tone of voice issue may be slightly difficult since lots of community members have put in a lot of time translating the existing strings.

Would be interested to hear what others think.

**Comment by alexgleason** [comment]
At first I wasn't unsure, but it's clear that you're not suggesting we force jokes in places where they're not needed. In the case of scheduling something in the future, MailChimp is effectively communicating what the problem is, and _why_. Saying "Please choose a date in the future" when you click submit isn't making an obvious statement to the content editor. They may not immediately understand what they did wrong, so the phrasing is effective here.

In other places, it could be a distraction, or make the messages _more_ difficult to understand. But I know you know that.

I think before we can proceed, we'll need to see some specific examples of what phrases could be changed to. I'm all for shifting to a more upbeat tone.

**Comment by heymonkeyriot** [comment]
> I think before we can proceed, we'll need to see some specific examples of what phrases could be changed to.

Agreed. But wanted to get some thoughts from others in relation to whether they think it's worth exploring.

>In other places, it could be a distraction, or make the messages more difficult to understand.

Also, obviously, agreed, we'll be doing damage to the UI if we put unnecessary friction in the way of the user, but if we can express something in a more friendly, personable way I think that will really enhance the user experience of the CMS.

**Comment by JoshBarr** [comment]
And then you'd need to translate all the strings, too.

**Comment by alexgleason** [comment]
I think it's worth exploring further. If it's only a change in tone and not meaning, I think we can safely find-and-replace-all those strings in the entire translations directory. Languages don't translate 1:1 anyway and I wouldn't be surprised if some of our translations have a different tone to their native audiences than our original phrasing anyway. The important thing is that it retains the same meaning. Besides, using a certain tone may be a part of localization for certain cultures of people. A good localization team will take this into account.

A part of me wishes we could connect a device to our brains and dump language strings into a "raw abstract ideas" binary format which is then localized, rather than English being the base. But maybe I'm getting off-topic here.

Basically, I don't see any harm in it. There's always room for improvement. Don't see the need to hold ourselves back if there's a better way.

**Comment by gasman** [comment]
Worth bearing in mind that "delighter" isn't necessarily synonymous with "jokey text" - it depends what we decide our tone / personality is going to be. A good example of an existing delighter is the Wagtail logo 'nodding' on hover.

Also, bear in mind there's a subsection of our audience who currently replace Wagtail's branding with their own (http://docs.wagtail.io/en/v1.3.1/advanced_topics/customisation/branding.html, #2353) - if they're branding-conscious enough to go to that trouble, they might not take too kindly to Wagtail going totally its own way when it comes to tone of voice. Obviously I wouldn't advocate us going down the bland corporate path on their behalf, but it's probably an argument against adopting full-on Innocent-Smoothie-speak :-)

**Comment by tomdyson** [comment]
I'm very keen on 'delighters' but I don't think we should inject any more personality to the copy. Wagtail is now being used in many different languages by airlines, universities, governments, banks, NGOs, schools, radio stations, charities, museums, startups, tech giants, publishers, churches, TV shows, gaming companies. I think think our tone has to be neutral, accurate and translatable, to accommodate the wide cultural range of editors / moderators / administrators within these sectors.

**Comment by heymonkeyriot** [comment]
Playing devil's advocate one could argue that Slack, and other SaaS products, have an equivalent user base. But I take the point from both @gasman and @tomdyson about the fact that Wagtail certainly shouldn't be getting in the way of the user.
